src/main/java/com/crowdin/client/ai/AiApi.java

```diff
@@ -0,0 +1,196 @@
+package com.crowdin.client.ai;
+
+import com.crowdin.client.ai.model.AddAiPromptRequest;
+import com.crowdin.client.ai.model.AddAiProviderRequest;
+import com.crowdin.client.ai.model.AiPrompt;
+import com.crowdin.client.ai.model.AiPromptResponseList;
+import com.crowdin.client.ai.model.AiPromptResponseObject;
+import com.crowdin.client.ai.model.AiProvider;
+import com.crowdin.client.ai.model.AiProviderModel;
+import com.crowdin.client.ai.model.AiProviderModelResponseList;
+import com.crowdin.client.ai.model.AiProviderResponseList;
+import com.crowdin.client.ai.model.AiProviderResponseObject;
+import com.crowdin.client.ai.model.AiProxyChatCompletionRequest;
+import com.crowdin.client.ai.model.AiProxyChatCompletionResponse;
+import com.crowdin.client.core.CrowdinApi;
+import com.crowdin.client.core.http.HttpRequestConfig;
+import com.crowdin.client.core.http.exceptions.HttpBadRequestException;
+import com.crowdin.client.core.http.exceptions.HttpException;
+import com.crowdin.client.core.model.ClientConfig;
+import com.crowdin.client.core.model.Credentials;
+import com.crowdin.client.core.model.PatchRequest;
+import com.crowdin.client.core.model.ResponseList;
+import com.crowdin.client.core.model.ResponseObject;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public class AiApi extends CrowdinApi {
+
+    public AiApi(Credentials credentials) {
+        super(credentials);
+    }
+
+    public AiApi(Credentials credentials, ClientConfig clientConfig) {
+        super(credentials, clientConfig);
+    }
+
+    /**
+     * @param userId User identifier
+     * @return list of Ai Prompts
+     * @see <ul>
+     * <li><a href="https://developer.crowdin.com/api/v2/#operation/api.ai.prompts.getMany" target="_blank"><b>API Documentation</b></a></li>
+     * </ul>
+     */
+    public ResponseList<AiPrompt> listAiPrompts (Long userId, Long projectId, String action, Long limit, Integer offset) throws HttpException, HttpBadRequestException {
+        Map<String, Optional<Object>> queryParams = HttpRequestConfig.buildUrlParams(
+                "projectId", Optional.ofNullable(projectId),
+                "action", Optional.ofNullable(action),
+                "limit", Optional.ofNullable(limit),
+                "offset", Optional.ofNullable(offset)
+        );
+        AiPromptResponseList aiPromptResponseList = this.httpClient.get(this.url + "/users/" + userId + "/ai/prompts", new HttpRequestConfig(queryParams), AiPromptResponseList.class);
```

Comment:
The Crowdin Enterprise version of these API endpoints has a different request URL (without the userId subpath)

https://developer.crowdin.com/enterprise/api/v2/#operation/api.ai.prompts.getMany

A similar situation existed in PR #242. Please consider the same approach in the current implementation.